tests/performance-tests/generate_test_config.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'yaml'
+
+manifest_file = ENV.fetch("CF_MANIFEST")
+manifest = YAML.load_file(manifest_file)
+
+admin_password = manifest.fetch("properties").fetch("performance_tests").fetch("admin_password")
+api_url = manifest.fetch("properties").fetch("cc").fetch("srv_api_uri")
```

Comment:
@combor and I talked about this and we've agreed to re-use the same config the moment. After further discussion with @alext we will de-dupe `generate_test_config.rb`/`run_test.sh`/`README.md`, but leave `init_test.go`.

We can review at a later date, when we have more configuration, whether it's useful to put additional config in the manifest or load a separate configuration file.